pkg/analyze/text_analyze_test.go

```diff
@@ -213,17 +306,34 @@ func Test_textAnalyze(t *testing.T) {
 			defer scopetest.End()
 			req := require.New(t)
 
-			getFiles := func(n string) ([]byte, error) {
-				val, ok := test.files[n]
-				if !ok {
+			getFiles := func(n string) (map[string][]byte, error) {
+				matching := make(map[string][]byte)
+				for k, v := range test.files {
+					if strings.HasPrefix(k, n) {
```

Comment:
I don't think this one is relevant

Reply:
@codelingobot reply @laverya nice catch! Next time you can just reply to the bot like this and it'll pop up on the rule chat. What minimum number of lines do you think we need within the if statement before wanting to avoid nesting? Also @marccampbell? Any thoughts on this (reply on the rule chat and I'll tighten up the rule linter accordingly).